mirdata/datasets/tiny_4mula.py

```diff
@@ -0,0 +1,319 @@
+"""4MuLA-tiny Dataset Loader
+
+.. admonition:: Dataset Info
+    :class: dropdown
+
+    This dataset is a tiny version extracted by 4MuLA dataset:
+
+    .. code-block:: latex
+
+        "4MuLA: A Multitask, Multimodal, and Multilingual Dataset of Music Lyrics and Audio Features."
+        by Angelo Cesar Mendes da Silva, Diego Furtado Silva, and Ricardo Marcondes Marcacini.
+        In Proceedings of the Brazilian Symposium on Multimedia and the Web (WebMedia '20). 2020.
+        DOI:https://doi.org/10.1145/3428658.3431089
+
+
+    The dataset consists of 1988 musics represents by melspectrogram and metadata. It
+    contains 27 genres and 93 artists. The audio file is not available. All attributes are:
+    ['music_id', 'music_name', 'music_lang', 'music_lyrics', 'art_id',
+    'art_name', 'art_rank', 'main_genre', 'related_genre', 'related_art',
```

Comment:
nit: edit suggestions, feel free to ignore

`1988 musics represents by -> 1988 music recordings represented by`
`It contains -> it conveys`
`The audio file is not available -> The audio files are not available`
`All attributes are -> The metadata includes the following attributes:`